src/cc_catalog_airflow/dags/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -0,0 +1,179 @@
+        raise Exception('Retries exceeded')
+
+    response = delayed_requester.get(
+        endpoint,
+        params=query_params,
+        timeout=60
+    )
+    if response is not None and response.status_code == 200:
+        try:
+            response_json = response.json()
+        except Exception as e:
+            logger.warning(f'Could not get response_json.\n{e}')
+            response_json = None
+
+    if (
+            response_json is None
+            or response_json.get('error') is not None
+    ):
+        logger.warning(f'Bad response_json:  {response_json}')
+        logger.warning(
+            'Retrying:\n_get_response_json(\n'
+            f'    {endpoint},\n'
+            f'    {query_params},\n'
+            f'    retries={retries - 1}'
+            ')'
+        )
+        response_json = _get_response_json(
+            query_params,
+            endpoint=endpoint,
+            retries=retries - 1
+        )
+
+    return response_json
+
+
+
+
+def _handle_the_response(response):
+    for data in response:
+        foreign_url = data.get('url', None)
+        single_image = data.get('images')
+        images_till_now = 0
+        if single_image is None:
+            continue
+        
+        image_url, key = _choose_image_version(single_image)
+        if image_url is None or key is None:
+            continue
+        width = single_image[key]['width']
+        height = single_image[key]['height']
+        foreign_id = data.get('id', image_url)
+        license_status = data.get('share_license_status', None).lower()
+        license_share = license_status.lower()
+        title = data.get('title', '')
+
+        creator_info = data.get('creators', {})
+        creator_name = None
+        if creator_info:
+            creator_name = creator_info[0].get('description', '')
+        meta_data = _create_meta_data(data)
+        images_till_now = image_store.add_item(
+            foreign_landing_url=foreign_url,  # Foreign Landing URl
+            image_url=image_url,  # Image URL
+            license_=license_status,  # License
+            license_version="1.0",  # License_Version
+            foreign_identifier=foreign_id,  # Foreign Identifier
+            width=width,  # Width of the image
+            height=height,  # height of the image
+            creator=creator_name,  # Creator Name
+            title=title,  # Title of the Image
+            meta_data=meta_data,  # Meta Data
+        )
+    return images_till_now
+
+
+def _choose_image_version(single_image):
+    if single_image.get('web'):
+        key = 'web'
+        image_url = single_image.get('web').get('url')
+    elif single_image.get('print'):
+        key = 'print'
+        image_url = single_image.get('print').get('url')
+    elif single_image.get('full'):
+        key = 'full'
+        image_url = single_image.get('full').get('url')
+    else:
+        key = None
+        image_url = None
```

Comment:
To address your question in the conversation, this is the correct behavior (to me) if there is no `'full'`: Just return `None, None`.  Good work!